blueprints/app/files/package.json

```diff
@@ -24,45 +24,45 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.4.2<% if (embroider) { %>",
-    "@embroider/compat": "^0.44.1",
-    "@embroider/core": "^0.44.1",
-    "@embroider/webpack": "^0.44.1<% } %>",
+    "@ember/test-helpers": "^2.6.0<% if (embroider) { %>",
+    "@embroider/compat": "^0.47.2",
+    "@embroider/core": "^0.47.2",
+    "@embroider/webpack": "^0.47.2<% } %>",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^2.2.0",
+    "ember-auto-import": "^2.2.4",
     "ember-cli": "~<%= emberCLIVersion %>",
     "ember-cli-app-version": "^5.0.0",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-htmlbars": "^6.0.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~4.0.0-beta.2",
+    "ember-data": "~4.0.0-beta.4",
     "ember-export-application-global": "^2.0.1",
     "ember-fetch": "^8.1.1",
     "ember-load-initializers": "^2.1.2",
     "ember-page-title": "^7.0.0",
-    "ember-qunit": "^5.1.4",
+    "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~4.0.0-beta.4",
-    "ember-template-lint": "^3.7.0<% if (welcome) { %>",
-    "ember-welcome-page": "^5.0.0<% } %>",
-    "eslint": "^7.32.0",
+    "ember-source": "~4.0.0-beta.10",
+    "ember-template-lint": "^3.14.0<% if (welcome) { %>",
+    "ember-welcome-page": "^6.0.0<% } %>",
+    "eslint": "^8.3.0",
```

Comment:
@kellyselden are we able to do this change yet? there were problems as it requires replacing deprecated `babel-eslint` with `@babel/eslint-parser`

Reply:
Thanks. I'll probably end up reverting it.

Reply:
done